invenio_theme/ext.py

```diff
@@ -85,44 +86,19 @@ def init_app(self, app, **kwargs):
         # Save reference to self on object
         app.extensions['invenio-theme'] = self
 
-    def init_config(self, config):
+    def init_config(self, app):
         """Initialize configuration.
 
-        :param config: A dict like object where default values should be set.
+        :param app: An instance of :class:`~flask.Flask`.
         """
-        config.setdefault('SASS_BIN', 'node-sass')
-        config.setdefault('THEME_SITENAME', 'Invenio')
-        config.setdefault('THEME_LOGO', 'images/invenio-color.svg')
-        config.setdefault('THEME_LOGO_ADMIN', 'images/invenio-white.svg')
-        config.setdefault('REQUIREJS_CONFIG', 'js/build.js')
-        config.setdefault('THEME_GOOGLE_SITE_VERIFICATION', [])
-        config.setdefault('BASE_TEMPLATE', 'invenio_theme/page.html')
-        config.setdefault(
-            'COVER_TEMPLATE', 'invenio_theme/page_cover.html')
-        config.setdefault(
-            'SETTINGS_TEMPLATE', 'invenio_theme/page_settings.html')
-        config.setdefault(
-            'HEADER_TEMPLATE', 'invenio_theme/header.html')
-        config.setdefault(
-            'THEME_BASE_TEMPLATE', config['BASE_TEMPLATE'])
-        config.setdefault(
-            'THEME_COVER_TEMPLATE', config['COVER_TEMPLATE'])
-        config.setdefault(
-            'THEME_SETTINGS_TEMPLATE', config['SETTINGS_TEMPLATE'])
-        config.setdefault(
-            'THEME_ERROR_TEMPLATE', 'invenio_theme/error.html')
-        config.setdefault(
-            'THEME_401_TEMPLATE', 'invenio_theme/401.html')
-        config.setdefault(
-            'THEME_403_TEMPLATE', 'invenio_theme/403.html')
-        config.setdefault(
-            'THEME_404_TEMPLATE', 'invenio_theme/404.html')
-        config.setdefault(
-            'THEME_500_TEMPLATE', 'invenio_theme/500.html')
-
-        config.setdefault('THEME_SEARCHBAR', True)
-        config.setdefault('THEME_SEARCH_ENDPOINT', '/search')
-        config.setdefault('THEME_BREADCRUMB_ROOT_ENDPOINT', '')
-        # Integration with Invenio-Admin:
-        config.setdefault(
-            'ADMIN_BASE_TEMPLATE', 'invenio_theme/admin.html')
+        for k in dir(config):
+            if k.startswith('THEME') or k.endswith('TEMPLATE') or k in [
```

Comment:
``THEME`` -> ``THEME_`` and I would go with:

``if k.startswith('THEME_') or k in [...explicit list of all non-conforming variables...]`` 

to make it very explicit which variables does't conform to the pattern.